plugin.js

```diff
@@ -19,6 +19,20 @@ module.exports = (opts = {}) => {
     return className.replace(/html ?\./, '')
   }
 
+  /**
+   * Process background value to remove unnecessary parts
+   * @param {string} value - background value
+   */
+  function processBackgroundValue(value) {
+    // check for image-set
+    // https://developer.mozilla.org/en-US/docs/Web/CSS/image/image-set
+    if (value.includes('image-set')) {
+      return /(image-set.*\))/gm.exec(value)[0] ?? value
+    } else {
+      return /(url.*\))/gm.exec(value)[0] ?? value
```

Comment:
Should we use proper parser here for the case of having `)` in file name? https://github.com/TrySound/postcss-value-parser

Reply:
Probably would be better, but it passed the test with `url(\'./image().png\') 50% 50%`

Reply:
Nice